drone-webdriver/src/test/java/org/jboss/arquillian/drone/webdriver/factory/ChromeHeadlessDriverTest.java

```diff
@@ -0,0 +1,52 @@
+package org.jboss.arquillian.drone.webdriver.factory;
+
+import java.io.IOException;
+import java.net.URL;
+import org.jboss.arquillian.drone.webdriver.configuration.WebDriverConfiguration;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ChromeHeadlessDriverTest {
+
+    @BeforeClass
+    public static void executeOnlyIfChromeOrChromeHeadlessBrowser() {
+        final String browser = System.getProperty("browser");
+        Assume.assumeTrue(browser.equals("chrome") || browser.equals("chromeHeadless"));
+    }
+
+    @Test
+    public void testOpenSimplePageUsingHeadlessChromium() throws IOException {
```

Comment:
if you name it `testOpenSimplePageUsingHeadlessChromium` then why you allow this test also for property `chrome`
in addition - it is not chromium but chrome.